src/pages/workspaces/workspace/Dashboard/WorkspaceDashboard.test.tsx

```diff
@@ -73,4 +73,14 @@ describe('WorkspaceDashboard', () => {
     // Assert
     expect(screen.getByText('test-description')).toBeInTheDocument();
   });
+
+  it('sets the the workspace description to open by default', () => {
+    // Arrange
+    // Act
+    render(<WorkspaceDashboard {...{ name, namespace, refreshWorkspace, workspace, storageDetails }} />);
+    
+    // Assert
+    expect(screen.getByText('Last Updated')).toBeInTheDocument(); //
```

Comment:
This would fail if the section is not open? I would think the HTML is still in the document, just not visible.

Reply:
Nevermind, I just tried it and looked at the HTML.